scopes/compositions/ui/composition-card/composition-card.tsx

```diff
@@ -23,11 +21,9 @@ export function CompositionCard({ Composition, name, link, className, ...rest }:
 
   return (
     <Card elevation="low" className={classNames(className, styles.compositionCard)} {...rest}>
```

Comment:
Can be removed from this PR

This is in backlog to implement. We will be re designing the error fallback screens.